docs/how-to-guide/how-to-guide.md

```diff
@@ -0,0 +1,358 @@
+# How-to guide
+
+### Configure WordPress\*
+
+\*This only works when setting up WordPress initially. Changing the value afterwards has no effect.
```

Comment:
This `initial_settings` configuration will only be effective if the WordPress haven't been initialized. Usually when you configure the db config or relate the wordpress charm with a database charm, it will be too late to set the `initial_setting`.

Reply:
Improved the sentence to clarify "initial", which is "before database relation setup".
```
\*This only works when setting up WordPress initially, before database relation setup. Changing the
value afterwards has no effect.
```
Thank you for the suggestion!